notebooks/official/tensorboard/tensorboard_profiler_custom_training_with_prebuilt_container.ipynb

```diff
@@ -45,22 +44,21 @@
         "    </a>\n",
         "  </td>\n",
         "  <td style=\"text-align: center\">\n",
+        "    <a href=\"https://console.cloud.google.com/vertex-ai/workbench/deploy-notebook?download_url=https://raw.githubusercontent.com/GoogleCloudPlatform/vertex-ai-samples/main/notebooks/official/tensorboard/tensorboard_profiler_custom_training_with_prebuilt_container.ipynb\">\n",
+        "      <img src=\"https://lh3.googleusercontent.com/UiNooY4LUgW_oTvpsNhPpQzsstV5W8F7rYgxgGBD85cWJoLmrOzhVs_ksK_vgx40SHs7jCqkTkCk=e14-rj-sc0xffffff-h130-w32\" alt=\"Vertex AI logo\"> <br>\n",
+        "      Open in Vertex AI Workbench\n",
+        "    </a>\n",
+        "  </td> \n",
+        "  <td style=\"text-align: center\">\n",
         "    <a href=\"https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/tensorboard/tensorboard_profiler_custom_training_with_prebuilt_container.ipynb\">\n",
```

Comment:
Where is the code for "[Open in Colab Enterprise]?? This should be added

Reply:
code for "[Open in Colab Enterprise] was already present. However, I missed changing the names for each logo as per notebook template guidelines. Made the edits accordingly and pushed the changes